Doc/library/secrets.rst

```diff
@@ -52,7 +52,7 @@ randomness that your operating system provides.
 
 .. function:: randbits(k)
 
-   Return an int with *k* random bits.
+   Returns a non-negative int with *k* random bits.
```

Comment:
```suggestion
   Return a non-negative int with *k* random bits.
```
All functions in the Python docs try to say "return" instead of "returns". Don't look at me - Guido started that decades ago :wink:.

Reply:
👀👀look